lib/pqa/mock_api_server.rb

```diff
@@ -0,0 +1,66 @@
+require 'sinatra/base'
+require 'date'
+
+module PQA
```

Comment:
Explain the purpose of this class and indicate that it is a development/testing harness